src/main/java/org/jenkinsci/plugins/pubsub/Message.java

```diff
@@ -69,10 +74,30 @@
  */
 public abstract class Message<T extends Message> extends Properties {
 
+    private static final Jenkins jenkins = Jenkins.getInstanceOrNull();
+    private static final String instanceIdentity;
+
+    static {
+        if (jenkins != null) {
+            InstanceIdentity identity = InstanceIdentity.get();
+            RSAPublicKey key = identity.getPublic();
+            instanceIdentity = new String(Base64.encodeBase64(key.getEncoded()), Charset.forName("UTF-8"));
```

Comment:
Take a look how we do this in the "Analytics" class in blueocean https://github.com/jenkinsci/blueocean-plugin/blob/57ac04497e59d04d75e35a0fe5984a42923ce1a6/blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/analytics/AbstractAnalytics.java#L58

The key is already sent on each response AFAIK, a shorter version might be appropriate given how frequently SSE events are sent? cc @vivek